test/util/nightwatch-commands/attestCheck.js

```diff
@@ -0,0 +1,38 @@
+import fs from 'fs';
+import path from 'path';
+import process from 'process';
+
+const ATTEST_PATH = './test/util/attest-deps/node_modules/attest/attest.js';
+const ATTEST_RULES_PATH = './test/util/attest-deps/node_modules/attest-rules/VA_508_ATtest.json';
+
+export function command(context, config, _callback) {
+  // Get source of the attest module and rules config
+  const attestSource = fs.readFileSync(path.resolve(process.cwd(), ATTEST_PATH), 'utf8');
+  const attestConfig = require(path.resolve(process.cwd(), ATTEST_RULES_PATH));
+
+  // Attach the attest source to the document
+  this.execute(innerAttestSource => {
+    const script = document.createElement('script');
+    script.text = innerAttestSource;
```

Comment:
@jkassemi correct but the code above assumes 2.1.6 which is not yet released as open source. It should be backwards compatible with 2.0.5 but you will want to validate that.